build.sbt

```diff
@@ -67,9 +69,16 @@ lazy val frontEnd = project("front-end")
       "org.webjars" % "react" % "0.14.3",
       "org.webjars" % "react-router" % "1.0.3",
       "org.webjars" % "jquery" % "2.2.0",
-      "org.webjars" % "foundation" % "5.3.0"
+      "org.webjars" % "foundation" % "5.3.0",
+      "com.typesafe.conductr" %% "lagom10-conductr-bundle-lib" % "1.4.0"
     ),
-    ReactJsKeys.sourceMapInline := true
+    ReactJsKeys.sourceMapInline := true,
+    // ConductR settings
+    BundleKeys.nrOfCpus := 1.0,
```

Comment:
I think that the correct place to do that would be in an `sbt-play-bundle` plugin - quite agnostic to Lagom.